internal/store/store.go

```diff
@@ -69,22 +91,54 @@ func newTransport(c config.Config) *digest.Transport {
 		Proxy:                 http.ProxyFromEnvironment,
 		IdleConnTimeout:       idleConnTimeout,
 		ExpectContinueTimeout: expectContinueTimeout,
+		TLSClientConfig:       tlsClientConfig,
+	}
+}
+
+func authenticatedClient(c config.Config) (*http.Client, error) {
+	t := &digest.Transport{
+		Username: c.PublicAPIKey(),
+		Password: c.PrivateAPIKey(),
+	}
+	if caCertificate := c.OpsManagerCACertificate(); caCertificate != "" {
+		dat, err := ioutil.ReadFile(caCertificate)
+		if err != nil {
+			return nil, err
+		}
+		fmt.Println(string(dat))
+		t.Transport = customCATransport(dat)
+	} else if skipVerify := c.OpsManagerSkipVerify(); skipVerify == yes {
+		t.Transport = skipVerifyTransport()
+	} else {
+		t.Transport = http.DefaultTransport
+	}
+
+	return t.Client()
+}
+
+func defaultClient(c config.Config) (*http.Client, error) {
+	client := http.DefaultClient
+	if caCertificate := c.OpsManagerCACertificate(); caCertificate != "" {
+		dat, err := ioutil.ReadFile(caCertificate)
+		if err != nil {
+			return nil, err
+		}
+		fmt.Println(string(dat))
```

Comment:
Was this used for dev/meant to be removed? Same with above on line 108?

Reply:
oops, thanks for the catch